releasenotes/notes/44002.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+issue:
+  - 44002
+releaseNotes:
+  - |
+    **Fixed** `istioctl x revision describe` error warning output.
```

```suggestion
    **Fixed** `istioctl experimental revision describe` warning gateway is not enabled when gateway exists.
```
Not sure if this is for both ingress and egress. Looking at the PR code it seems to be both.

---

Yes I think it's both, the suggestion looks great.